Examples/Objective-C/ViewController.m

```diff
@@ -63,7 +63,7 @@ - (void)suspendNotifications {
 }
 
 - (void)alertLevelDidChange:(NSNotification *)notification {
-    MBRouteProgress *routeProgress = (MBRouteProgress *)notification.userInfo[MBRouteControllerNotificationProgressDidChange];
+    MBRouteProgress *routeProgress = (MBRouteProgress *)notification.userInfo[MBRouteControllerProgressDidChangeNotificationProgressKey];
```

Comment:
This incorrect notification caused the resulting string to be something like "In 0mm (null)"